app/views/transactions/new.html.erb

```diff
@@ -1,11 +1,12 @@
-<div class="title">
+ <div class="title">
   <%= link_to home_index_path, class: 'arrow' do %>
     <i class="fa-solid fa-chevron-left"></i>
   <% end %>
-  <h3>NEW TRANSACTION</h3>
+  <h3 style="margin-left: 20%">NEW TRANSACTION</h3>
 </div>
 <%= form_with model: @transaction, url: transactions_create_path, html: { class: 'form' } do |f| %>
   <%= f.text_field :name, placeholder: 'Transaction' %>
   <%= f.number_field :amount, step: "0.01", placeholder: 'Amount' %>
-  <%= f.submit "Submit", class: 'category-new-btn' %>
+  <%= f.text_field :name :category, placeholder: 'Category' %>
```

Comment:
- [ ] [REQUIRED] Hello @banzaman , you have done a good job so far, however, we need to ensure that a user can select from the list on the available categories to add a transaction to. Let's remove the numbers increment on the category field.
<img width="1437" alt="Screenshot 2024-03-03 at 08 02 54" src="https://github.com/banzaman/budget_app/assets/58692590/77408798-3e6b-482d-b9d3-b6d9492baddb">